README.md

```diff
@@ -37,13 +37,32 @@ OpenSearch Dashboards Reports integration with [Notifications](https://github.co
 
 ## Troubleshooting
 
-#### Fail to launch Chromium
+### Fail to launch Chromium
 
 There could be two reasons for this problem
 
 1. You are not having the correct version of headless-chrome matching to the OS that your OpenSearch Dashboards is running. Different versions of headless-chrome can be found [here](https://github.com/opensearch-project/dashboards-reports/releases/tag/chromium-1.12.0.0)
 
 2. Missing additional dependencies. Please refer to [additional dependencies section](./dashboards-reports/rendering-engine/headless-chrome/README.md#additional-libaries) to install required dependencies according to your operating system.
+
+### System Dependencies
```

Comment:
Since this title is under `Troubleshooting` section, I think it's better for title to be clear about the `trouble`, something like `Fail to/couldn't ...`. Also we need to update the documentation on our Opensearch website. Do you know if there's any related tracking issue opened for that?

Reply:
Sure we can have a more specific title. No related tracking issue for OpenSearch documentation but offline I am planning to open this issue with Infra team to have these types of dependencies auto-installed.